<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="frame" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#frame{
    
	background-color: qlineargradient(spread:pad, x1:0.467, y1:0.37, x2:1, y2:1, stop:0 rgba(0, 80, 138, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20 20pt &quot;Sitka Heading Semibold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Make your unforgettable train experience with us!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectTrainButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>710</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background-color: rgb(255, 255, 255);
font: 63 12pt &quot;Sitka Small Semibold&quot;;
color:rgb(0, 86, 148);
}

QPushButton:hover{
	background-color: #00508a;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Select Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>780</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 20px;
background-color:qlineargradient(spread:pad, x1:0.467, y1:0.37, x2:1, y2:1, stop:0 rgba(0, 80, 138, 255), stop:1 rgba(255, 255, 255, 255));
font: 63 12pt &quot;Sitka Small Semibold&quot;;
color:white;
}

QPushButton:hover{
	background-color: white;
	color: #00508a;
}</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>700</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: 8pt &quot;Sitka Small Semibold&quot;;
color:rgb(0, 86, 148);
border: 2px solid white;
border-radius: 7px;</string>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>126</number>
    </attribute>
    <column>
     <property name="text">
      <string>Train ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Capacity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Number of cart</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
